docker-compose-production.yml

```diff
@@ -77,3 +77,28 @@ services:
       context: ./
       dockerfile: docker/prod/code-upload-worker/Dockerfile
 
+  prometheus:
```

Comment:
@burnerlee let's add this only for staging deployment for now. Once we have a working setup we can deploy it on production